### PR TITLE
Visual tweaks

### PR DESCRIPTION
**Pull request by cecileboucheron**

### CODE REVIEW

File: shared/profile/friendships.desktop.js [hunk 1]
```diff
@@ -108,13 +108,11 @@ class FriendshipsRender extends Component<Props> {
             this.props.onSwitchTab && this.props.onSwitchTab('Followers')
           }}
         >
-          <Box style={{marginTop: globalMargins.small}}>
+          <Box style={{marginTop: globalMargins.large}}>
```
Comment (buoyad) [inline shared/profile/friendships.desktop.js]: This also applies to the friendship list when it's not empty, is that intended?

image

File: shared/profile/friendships.desktop.js [hunk 2]
```diff
@@ -132,13 +130,11 @@ class FriendshipsRender extends Component<Props> {
             this.props.onSwitchTab && this.props.onSwitchTab('Following')
           }}
         >
-          <Box style={{marginTop: globalMargins.small}}>
+          <Box style={{marginTop: globalMargins.large}}>
```
Comment (buoyad) [inline shared/profile/friendships.desktop.js]: same here

File: shared/tracker/index.desktop.js [hunk 3]
```diff
@@ -106,7 +106,9 @@ export default class TrackerRender extends PureComponent<RenderProps> {
                     style={{
                       ...globalStyles.flexBoxRow,
                       alignItems: 'flex-start',
+                      cursor: 'pointer',
                       justifyContent: 'flex-start',
+                      marginBottom: globalMargins.xtiny,
```
Comment (buoyad) [inline shared/tracker/index.desktop.js]: I'm not sure if this line is what's causing it, but if you scroll down on a tracker the background color shows through briefly:
tracker_-_mikem_and_developer_tools_-_file____users_dan_go_src_github_com_keybase_client_shared_desktop_renderer_renderer_html_mainwindow_react_perf

Doesn't appear on master.

File: shared/profile/friendships.desktop.js (hunk 1 above)
Comment (cecileboucheron) [reply shared/profile/friendships.desktop.js]: Woops, nope.

**Comment by cecileboucheron** [comment]
Ok just fixed everything (git icon will be on desktop only until we make zIndex work) + tweaked one or 2 things.

**Pull request merged**